### ISSUE TITLE
OBS cannot map DS4 triggers  

### ISSUE THREAD

**Issue by Squadrani** [univrsal/input-overlay#253]
Hi. I want to map the triggers of my DS4. At the provided graphical tool the plugin can recognize the input of the triggers (of all buttons to be more exactly), but at OBS it can't. I had problems with mapping the buttons too, but i've found a solution with the .json at this answer https://github.com/univrsal/input-overlay/issues/175#issuecomment-909778106 . I had to change my keycodes of the buttons to turn them similar to the answer, but i can't do the same with the triggers. I tried to change the parameters, but without success.

I'll let the .png and the .json on a zip file

[Overlay.zip](https://github.com/univrsal/input-overlay/files/8792914/Overlay.zip)

Plugin version: 5.0.0

OBS log: https://obsproject.com/logs/hLz2f_tZwcF1zofs

**Comment by HereInPlainSight** [comment]
I... cannot confirm this.  I have a DS4 controller, and I actually have a video you can see of me using the triggers over in [this comment](https://github.com/univrsal/input-overlay/issues/241#issuecomment-1140127954) using the default dualsense.json in both the CCT and in OBS itself (and your .json file does not differ significantly in regards to the triggers).  I tested your overlay.zip and both triggers seem to work for me.

I'm assuming we have different models of DS4 controllers that differ in significant ways (since the fix worked for your arrow keys but isn't even a starter for mine).  If you can download the [nightly.7z from libgamepad](https://github.com/univrsal/libgamepad/releases/tag/v1.1) (you're on 64-bit Windows, so I believe you'll want the `x86_64-win32-msvc/bin/libgamepad_tests.exe` file in the archive) and provide a log of the output of *just* your trigger keys (along with the order you pressed them), that would probably be more helpful to the dev for figuring out what's going on.

**Comment by dekuscrublife** [comment]
I've been having the same problem as OP. I was going to try using libgamepad and post my log, but the program has been sitting for around 10 minutes and doesn't seem to realize I have a controller plugged in. It just keeps saying "debug: Updating device list" over and over. I tried using the Gamepad Bindings menu in the input_overlay settings in OBS but each trigger has 3 values depending on how far the triggers are pressed in, and none of the three seem to work. I've tried disabling trigger mode in the .json file, but that didn't seem to help either.
